config/locales/sk.yml

```diff
@@ -40,18 +40,18 @@ sk:
     sidebars:
       filters: "Filtre"
     pagination:
-      empty: "Nenájdený."
-      one: "Zobrazená  <b>1</b> položka"
-      one_page: "Počet zobrazených položiek %{n}"
-      multiple: "<b>%{from}&nbsp;-&nbsp;%{to}</b> z <b>%{total}</b>"
-      multiple_without_total: "<b>%{from}&nbsp;-&nbsp;%{to}</b>"
+      empty: "Nenájdený %{model}."
+      one: "Zobrazená  <b>1</b> %{model}"
+      one_page: "Počet zobrazených %{model} %{n}"
```

Comment:
An inflection should be used (plural genitiv)

Reply:
You mean there are different pluralization for nouns in Slovakian, according to the rule they play in the sentence? I think we only support `model` or `plural_model`... :disappointed: